packages/components/BackButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { IconTextButton } from "./IconTextButton";
+import { Left, Right } from "./svg";
+import "./BackButton.css";
+
+function BackButton() {
+	return (
+		<div className="backButton">
+			<IconTextButton
+				text="Back"
+				href="javascript:window.history.back();"
```

Comment:
You should be using the React Router `useHistory()` hook, weird javascript isn't great

Reply:
Moved into widgets and replaced with useNavigate